src/mongocxx/include/mongocxx/v1/database.hpp

```diff
@@ -31,7 +54,296 @@ namespace v1 {
+    ///
+    /// Set the default "readPreference" to use for operations on this database.
+    ///
+    /// Overrides the default inherited from the associated client. May be overridden by individual operations.
+    ///
+    /// @see
+    /// - [Default MongoDB Read Concerns/Write Concerns (MongoDB Manual)](https://www.mongodb.com/docs/manual/reference/mongodb-defaults/)
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(void) read_preference(v1::read_preference rp);
+
+    ///
+    /// Return the current default "readPreference" to use for operations on this database.
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(v1::read_preference) read_preference() const;
+
+    ///
+    /// Set the default "writeConcern" to use for operations on this database.
+    ///
+    /// Overrides the default inherited from the associated client. May be overridden by individual operations.
+    ///
+    /// @see
+    /// - [Default MongoDB Read Concerns/Write Concerns (MongoDB Manual)](https://www.mongodb.com/docs/manual/reference/mongodb-defaults/)
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(void) write_concern(v1::write_concern wc);
+
+    ///
+    /// Return the current default "writeConcern" to use for operations on this database.
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(v1::write_concern) write_concern() const;
+
+    ///
+    /// Access the collection with the given name.
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(v1::collection) collection(bsoncxx::v1::stdx::string_view name) const;
+
+    ///
+    /// Equivalent to `this->collection(name)`.
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(v1::collection) operator[](bsoncxx::v1::stdx::string_view name) const;
+
+    ///
+    /// Return a GridFS bucket for this database.
+    ///
+    /// @note When the "bucketName" field is unset, the default bucket name "fs" is used instead.
+    /// @note When the "chunkSizeBytes" field is unset, the default chunk size of 255 kB is used instead.
+    ///
+    /// @see
+    /// - [GridFS (MongoDB Manual)](https://www.mongodb.com/docs/manual/core/gridfs/)
+    ///
+    MONGOCXX_ABI_EXPORT_CDECL(v1::gridfs::bucket) gridfs_bucket(v1::gridfs::bucket::options const& opts = {}) const;
+
+    ///
+    /// Return a change stream subscribed to this database.
+    ///
+    /// @see
+    /// - [Change Streams (MongoDB Manual)](https://www.mongodb.com/docs/manual/changeStreams/)
+    ///
+    /// @{
+    MONGOCXX_ABI_EXPORT_CDECL(v1::change_stream) watch(v1::change_stream::options const& opts = {});
+
+    MONGOCXX_ABI_EXPORT_CDECL(v1::change_stream) watch(
+        v1::client_session const& session,
+        v1::change_stream::options const& opts = {});
+    /// @}
+    ///
+
+    ///
+    /// Return a change stream subscribed to this collection with events filtered/modified by `pipeline`.
+    ///
+    /// @see
+    /// - [Change Streams (MongoDB Manual)](https://www.mongodb.com/docs/manual/changeStreams/)
+    ///
+    /// @{
+    MONGOCXX_ABI_EXPORT_CDECL(v1::change_stream) watch(
+        v1::pipeline const& pipe,
```

Comment:
Rename `pipe` to `pipeline` to match doc comment. Also applies to overload below.